docs/git-commands.md

```diff
@@ -0,0 +1,101 @@
+# Git Hub Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo. This copy of the repo will have the remote name of `origin`.
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
+
+It should also be noted you will need to have forked the repo, how to do this can be seen in [our tutorial](Tutorial.md)
+
+```
+git remote add <<your-name>> https://github.com/<<Your Github UserName Here>>/SchoolWebsite.git
+```
+
+## Committing and Creating PR's
+So now you have following completed:
+- The repo cloned
+- Your fork remote added
+- The main repo remote added
+
+You can now begin to work and create your own PR against the main repo. To do this, you must create a branch. It is a good convention to give the branch a descriptive name, usually something descibing the work in the PR, keep this name short and too the point, example `add-contact-page`
```

Comment:
Could be useful to quickly describe what a branch is/what creating it does, and what a PR is?